packages/theme/src/button/button.css.ts

```diff
@@ -102,23 +105,22 @@ export const button = recipe({
       square: style([
```

Comment:
this style was unused before but it was intended to be used for the square buttons, like we see in app header and drawer etc.